modules/custom/social_gdpr/social_gdpr.install

```diff
@@ -61,6 +61,10 @@ function _social_gdpr_get_permissions($role) {
     'access data policy revisions',
     'revert all data policy revisions',
     'overview user consents',
+    'edit inform and consent setting',
+    'overview inform and consent settings',
+    'Administer inform and consent settings',
```

Comment:
This should be all lowercase (otherwise strict checks might say it's a different string).

e.g. `var_dump("one" === "One"); // false`